doc/etcd-defrag-cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: etcd-defrag
+  namespace: kube-system
+spec:
+  schedule: "14 9 * * 1-5"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: etcd-defrag
+            image: ghcr.io/ahrtr/etcd-defrag:v0.4.0
+            args:
+            - --cacert=/etc/kubernetes/pki/etcd/ca.crt
+            - --cert=/etc/kubernetes/pki/etcd/peer.crt
+            - --key=/etc/kubernetes/pki/etcd/peer.key
+            - --cluster
+            - --defrag-rule
+            - "dbQuotaUsage > 0.8 || dbSize - dbSizeInUse > 200*1024*1024"
+            volumeMounts:
+            - mountPath: /etc/kubernetes/pki/etcd
+              name: etcd-certs
+          volumes:
+          - hostPath:
+              path: /etc/kubernetes/pki/etcd
+              type: Directory
+            name: etcd-certs
+          restartPolicy: OnFailure
+          hostNetwork: true
```

Comment:
If etcd runs as a POD, then it seems there is no need to use hostNetwork.

Reply:
`hostNetwork` is needed to reach the apiserver with `127.0.0.1:2379`. I will add it as a comment. 
`hostNetwork` could be removed if the actual IP address endpoint of etcd is used.
again a conveniency/pragmatism story.